test/acceptance/pages/select-business-page.js

```diff
@@ -0,0 +1,159 @@
+const CommonActions = require('./common-actions')
+const getMagicLink = require('../support/get-magic-link')
+
+// select business element
+const BUSINESS_EMAIL = 'ibrahim.adekanmi@kainos.com'
+const RPA_CONTACT = '.govuk-details'
+const PAGE_TITLE = 'Annual health and welfare review of livestock'
+const SELECT_BUSINESS = 'Which business'
+const BUSINESS_NAME = 'Ibrahim Farm'
+const CONTACT = 'Telephone'
+const BUSINESS_CHECK_BUTTON = '#whichBusiness'
+const NO_BUSINESS_CONTENT = '.govuk-details__text'
+const BUSINESS_LIST = '[for="whichBusiness"]'
+// org-review element
+const CHECK_DETAILS = '.govuk-heading-l'
+const FARMER_DETAILS = '.govuk-summary-list'
+const DETAILS_BUTTON = '#confirmCheckDetails'
+const CONTINUE_BUTTON = '#btnContinue'
+const DETAILS = 'Check your details'
+const CONTENT = 'Farmer name'
+const REVIEW = 'Which livestock do you want a review for?'
+//select livestock page
+const WHICH_REVIEW = '.govuk-fieldset__heading'
+const LIVESTOCK_TYPE = '[data-module="govuk-radios"]'
+const SELECT_ANIMAL = '#whichReview-3'
+const LIVESTOCK = 'Sheep'
+//eligibility page
+const REQUIRE_LIVESTOCK_NUMBER = '#eligibleSpecies-hint'
+const CONFIRM_ELIGIBILITY = '#eligibleSpecies'
+const DECLARATION = '[role="button"]'
+const TERMS_CONDITIONS = '[href="/apply/terms"]'
+const TERMS_AND_CONDITION_BOX = '#terms'
+const COMPLETE_APPLICATION = '[value="accepted"]'
+const SUCCESS_MESSAGE = '.govuk-panel__title'
+const ACCURATE_ANSWER = 'Check your answers'
+const AGREED = 'declaration'
+const REVIEW_AGREED = 'agreement'
+const TERMS = 'Accept the terms'
+const MESSAGE = 'Application complete'
+const LIVESTOCK_NUMBER = 'eligible for funding'
+
+class SelectBusinessPage extends CommonActions {
+  async magicLinkUrl () {
+    const sleep = (waitTimeInMs) => new Promise(resolve => setTimeout(resolve, waitTimeInMs))
+    console.log('sleeping')
```

Comment:
can remove logs now that we know it works

Reply:
removed